app/scripts/migrations/119.test.ts

```diff
@@ -0,0 +1,93 @@
+import { createMockInternalAccount } from '../../../test/jest/mocks';
+import { migrate, version } from './119';
+
+const oldVersion = 118;
+const newVersion = 119;
+
+describe('migration #119', () => {
+  it('should update the version metadata', async () => {
+    const oldStorage = {
+      meta: { version: oldVersion },
+      data: {},
+    };
+
+    const newStorage = await migrate(oldStorage);
+    expect(newStorage.meta).toStrictEqual({ version });
+  });
+
+  it('should add importTime to InternalAccount if it is not defined"', async () => {
+    const mockInternalAccount = createMockInternalAccount();
+    // @ts-expect-error forcing the importTime to be undefined for migration test.
+    mockInternalAccount.metadata.importTime = undefined;
+
+    const oldStorage = {
+      meta: {
+        version: oldVersion,
+      },
+      data: {
+        AccountsController: {
+          internalAccounts: {
+            accounts: {
+              [mockInternalAccount.id]: mockInternalAccount,
+            },
+            selectedAccount: mockInternalAccount.id,
+          },
+        },
+      },
+    };
+
+    const newStorage = await migrate(oldStorage);
+    expect(newStorage).toStrictEqual({
+      meta: {
+        version: newVersion,
+      },
+      data: {
+        AccountsController: {
+          internalAccounts: {
+            accounts: {
+              [mockInternalAccount.id]: {
+                ...mockInternalAccount,
+                metadata: {
+                  ...mockInternalAccount.metadata,
+                  importTime: expect.any(Number),
```

Comment:
Do you think it would add value to this test to mock the `Date` and check that the number is correct?

```ts
const mockTimestamp = ... ;

const getStubbedDate = (): number => new Date(MOCK_TIMESTAMP).getTime();

jest.spyOn(global.Date, 'now').mockImplementation(() => getStubbedDate());
```

Reply:
Perhaps we can use the build-in Jest fake timers:

```typescript
jest
  .useFakeTimers()
  .setSystemTime(new Date(MOCK_TIMESTAMP));
```